pandas/tests/io/excel/test_openpyxl.py

```diff
@@ -396,3 +401,38 @@ def test_ints_spelled_with_decimals(datapath, ext):
     result = pd.read_excel(path)
     expected = DataFrame(range(2, 12), columns=[1])
     tm.assert_frame_equal(result, expected)
+
+
+def test_write_date_datetime_format(ext):
+    # see gh-44284
+    #
+    # Test that custom date/datetime formats are respected
```

Comment:
Please remove these comments, you only need the gh reference

Reply:
Amended in commit 5362737.